frameworks/PHP/openswoole/benchmark_config.json

```diff
@@ -14,34 +14,13 @@
       "database": "MySQL",
       "framework": "openswoole",
       "language": "PHP",
-      "flavor": "PHP8.1",
+      "flavor": "PHP8.2",
       "orm": "Raw",
       "platform": "openswoole",
       "webserver": "none",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "Open Swoole",
-      "notes": "",
-      "versus": "php"
-    },
-    "postgres": {
```

Comment:
Postgres variant is deleted in config.json.
But you created a open swoole postgre server, and never will be executed.

Please add again the postgres variant here.